examples/data/stream-data.json

```diff
@@ -0,0 +1,35 @@
+[
```

Comment:
Rather than a raw `.json` file we should create an `input` event generator that randomizes certain properties. This function could live in the examples `utils.js` module mentioned elsewhere.